Foundation/NSKeyedArchiver.swift

```diff
@@ -244,7 +244,7 @@ open class NSKeyedArchiver : NSCoder {
         if objv != nil &&
             self.requiresSecureCoding &&
             !NSKeyedArchiver._supportsSecureCoding(objv) {
-            fatalError("Secure coding required when encoding \(objv)")
```

Comment:
You should change this to bind `obj` in an `if-let` and remove the force unwrap.